package.json

```diff
@@ -39,70 +39,70 @@
     "cypress:open": "cypress open"
   },
   "devDependencies": {
-    "@testing-library/jest-dom": "^5.11.9",
+    "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^11.2.5",
     "@testing-library/user-event": "^12.8.3",
     "@types/jest": "^26.0.20",
     "@typescript-eslint/eslint-plugin": "^4.15.1",
     "@typescript-eslint/parser": "^4.15.1",
     "cypress": "^6.5.0",
     "eslint-import-resolver-typescript": "^2.4.0",
-    "eslint-plugin-import": "^2.22.1",
-    "eslint-plugin-jsx-a11y": "^6.4.1",
-    "eslint-plugin-react": "^7.22.0",
-    "mock-socket": "^9.0.3",
-    "node-sass": "^5.0.0",
-    "prettier": "^2.2.1",
-    "typescript": "^4.2.3"
+    "eslint-plugin-import": "^2.26.0",
+    "eslint-plugin-jsx-a11y": "^6.6.1",
+    "eslint-plugin-react": "^7.31.10",
+    "mock-socket": "^9.1.5",
+    "prettier": "^2.7.1",
```

Comment:
i would have split prettier update and code formatting in its own commit.
I think it's still easily achievable by updating prettier and formatting the codebase and then rebasing the current PR ontop of it